src/renderer/components/SettingsModal.tsx

```diff
@@ -206,6 +208,25 @@ const AppearanceSettings = memo(() => {
                     </NumberInputStepper>
                 </NumberInput>
             </SettingsItem>
+
+            <SettingsItem
+                description="The amount of padding for the viewport PNG export."
+                title="Viewport PNG export padding"
+            >
+                <NumberInput
+                    defaultValue={viewportExportPadding || 20}
+                    max={100}
+                    min={0}
+                    value={Number(viewportExportPadding || 0)}
+                    onChange={(number: string) => setViewportExportPadding(Number(number || 0))}
```

Comment:
This will set the value to NaN if the current input is not a valid number.

(My suggestion still needs some code formatting.)
```suggestion
                    onChange={(number: string) => {
                       const value = Number(number)
                       if (!Number.isNaN(value)) {
                           setViewportExportPadding(value)
                       }
                    }}
```